Update quickstart-sms.md (#2226)
semgrep/semgrep-docs: docs/getting-started/quickstart-sms.md

#### Bitbucket Data Center

- Server/Data Center 8.8+: Project-level webhooks available to support Diff-aware scanning in Semgrep Managed Scans

> - For Server/Data Center less than 8.8 please contact your Semgrep Account Manager or [Semgrep Support](https://semgrep.dev/docs/support)

Reviewer: ```suggestion
- Version 8.7 and earlier: contact your Semgrep Account Manager or [Support](/support)
```

> For Bitbucket Server/Data Center less than 7.18, you must provide a [Personal Access Token](https://confluence.atlassian.com/bitbucketserver0717/personal-access-tokens-1087535496.html) to Semgrep, with 'PROJECT_ADMIN' permissions.

Reviewer: ```suggestion
For Bitbucket Server or Data Center version 7.18 or earlier, you must provide a [Personal Access Token](https://confluence.atlassian.com/bitbucketserver0717/personal-access-tokens-1087535496.html) with 'PROJECT_ADMIN' permissions to Semgrep
```
